tests/stdlib/tmath.nim

```diff
@@ -270,6 +270,22 @@ block:
       doAssert floorMod(8.0, -3.0) ==~ -1.0
       doAssert floorMod(-8.5, 3.0) ==~ 0.5
 
+    block: # euclDiv/euclMod
+      doAssert euclDiv(8, 3) == 2
+      doAssert euclMod(8, 3) == 2
+
+      doAssert euclDiv(8, -3) == -2
+      doAssert euclMod(8, -3) == 2
+
+      doAssert euclDiv(-8, 3) == -3
+      doAssert euclMod(-8, 3) == 1
+
+      doAssert euclDiv(-8, -3) == 3
+      doAssert euclMod(-8, -3) == 1
+
+      doAssert euclMod(8.0, -3.0) ==~ 2.0
```

Comment:
ok.
side: note: we should in future work replace
`proc `==~`(x, y: float): bool = (abs(x-y) < 1e-9)`
by the new `almostEqual`

Reply:
I will clean them later